packages/datagateway-common/src/homePage/homePage.component.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { createMount, createShallow } from '@material-ui/core/test-utils';
+import { HomePageWithoutStyles, HomePageProps } from './homePage.component';
+
+describe('Home page component', () => {
+  let shallow;
+  let mount;
+  let props: HomePageProps;
+
+  beforeEach(() => {
+    shallow = createShallow({ untilSelector: 'div' });
+    mount = createMount();
```

Comment:
`mount` is not needed as the test uses `shallow`. This also means that the `afterEach` block can also be removed.